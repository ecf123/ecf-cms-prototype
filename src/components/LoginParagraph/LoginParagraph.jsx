import React from 'react'
import './LoginParagraph.scss'

const LoginParagraph = () => {
  return (
    <div>
        <p className='title__text'>Feugiat magna viverra pellentesque habitant. Libero id amet nisl, tellus cras eleifend et in. </p>
    </div>
  )
}

export default LoginParagraph
