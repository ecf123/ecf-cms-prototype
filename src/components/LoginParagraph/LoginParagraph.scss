@use "../../styles/variables.scss" as *;
@use "../../styles/typography.scss" as *;

.title__text {
    color: $color-grey-dark;
    font-family: $font-secondary;
    font-size: 14px;
    font-weight: 400;
    line-height: 160%;
    margin-bottom:30px;
}