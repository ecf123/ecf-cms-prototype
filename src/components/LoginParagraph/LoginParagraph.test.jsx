import { customRender } from "../../utils/testUtils";
import LoginParagraph from "./LoginParagraph";

it("should render the paragraph", () => {
    const { container } = customRender(
    <LoginParagraph>
        <p>Feugiat magna viverra pellentesque habitant. Libero id amet nisl, tellus cras eleifend et in. </p>
    </LoginParagraph>
    )
    expect (container).toMatchSnapshot();
}
)