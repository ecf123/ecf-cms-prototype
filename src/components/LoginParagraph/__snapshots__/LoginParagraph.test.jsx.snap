// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the paragraph 1`] = `
<div>
  <div>
    <p
      class="title__text"
    >
      Feugiat magna viverra pellentesque habitant. Libero id amet nisl, tellus cras eleifend et in. 
    </p>
  </div>
</div>
`;
